test/Week1Spec.hs

```diff
@@ -0,0 +1,16 @@
+module Week1Spec (spec) where
+
+import Week1.HW01 
+import Test.Hspec
+
+spec :: Spec
+spec =
+    describe "Week 1" $ do
+        describe "lastDigit" $ do
+            it "should return last digit for a number" $
+                lastDigit 15 `shouldBe` 5
```

Comment:
Using [shouldBe](https://hackage.haskell.org/package/hspec-expectations-0.8.2/docs/Test-Hspec-Expectations.html#v:shouldBe) and friends will save you some from having to create your own test framework.